src/actions/channels.js

```diff
@@ -808,6 +808,37 @@ export function getChannels(teamId, page = 0, perPage = General.CHANNELS_CHUNK_S
     };
 }
 
+export function autocompleteChannels(teamId, term) {
```

Comment:
Can you add a unit test for this action?

Reply:
💩 You caught me. I went to copy and paste the "searchChannels" unit test, saw that there wasn't one, and thought "This looks hard...".